django/contrib/postgres/expressions.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db.models import Subquery
+
+
+class ArraySubquery(Subquery):
+    template = 'ARRAY(%(subquery)s)'
+
+    def __init__(self, queryset, **kwargs):
+        super().__init__(queryset, **kwargs)
```

Comment:
I could be missing something here but is this even necessary?

```suggestion
```

Reply:
My reasoning for this override was to hide `output_field` from the signature since setting through `__init__` it is a no-op due to the property-override.